j2k/testData/fileOrElement/kotlinApiAccess/ObjectMembers.kt

```diff
@@ -4,8 +4,8 @@ internal class C {
     fun foo(): Int {
         KotlinObject.property1 = 1
         KotlinObject.property2 = 2
-        return KotlinObject.foo() +
+        return (KotlinObject.foo() +
```

Comment:
The parentheses are only required if the line break is before the operator. In this case they should not be added.